tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -249,6 +250,19 @@ void GetInputProperties(const grappler::GraphProperties& graph_properties,
   }
 }
 
+// This function checks if a tensor is compatible with TRT.
+//
+// We check that the shape and datatype is compatible with TensorRT. We also
+// return the corresponding trt_dtype, the trt_dims and the batch_size (latter
+// is only needed in implicit batch mode).
+//
+// The return status indicates wether the tensor is compatible.
+//
+// If validation_only == false, then we make an additional check. In implicit
```

Comment:
Can we simplify this paragraph?
For implicit batch mode, when validation_only == false, we also check that all input dimensions besides the implicit batch dimension, are known dimensions.